test_examples.py::test_run_example_notebook fail because no Jupyter kernels

Probably due to change in `jupyter` package that doesn't automatically register 'python3' default kernel.

As seen in the CI logs of the recent PR #286: https://app.circleci.com/pipelines/github/HumanCompatibleAI/imitation/786/workflows/1c42cd18-519f-4ad3-afd5-82abebde6211/jobs/1631

```
=================================== FAILURES ===================================
_____________________ test_run_example_notebooks[nb_path0] _____________________
[gw7] linux -- Python 3.7.5 /venv/bin/python3.7

nb_path = PosixPath('/imitation/tests/../examples/density_baseline_demo.ipynb')

    @pytest.mark.parametrize("nb_path", NB_PATHS)
    def test_run_example_notebooks(nb_path):
        """Smoke test ensuring that example notebooks run without error.
    
        The `pytest_notebook` package also includes regression test functionality against
        saved notebook outputs, if we want to check that later.
        """
        nb = notebook.load_notebook(nb_path)
>       execution.execute_notebook(nb, cwd=EXAMPLES_DIR, timeout=120)

tests/test_examples.py:23: 
```

--- a/setup.py
+++ b/setup.py
@@ -69,6 +69,7 @@ setup(
             "ntfy[slack]",
             "ipdb",
             "isort~=5.0",
+            "ipykernel",
             "jupyter",
             "pytype",
             "codespell",

--- a/setup.py
+++ b/setup.py
@@ -14,6 +14,8 @@ TESTS_REQUIRE = [
     "flake8-builtins",
     "flake8-debugger",
     "flake8-isort",
+    "ipykernel",
+    "jupyter",
     "pytest",
     "pytest-cov",
     "pytest-notebook",
@@ -69,8 +71,6 @@ setup(
             "ntfy[slack]",
             "ipdb",
             "isort~=5.0",
-            "ipykernel",
-            "jupyter",
             "pytype",
             "codespell",
             # for convenience

--- a/src/imitation/analysis/mountain_car_plots.py
+++ b/src/imitation/analysis/mountain_car_plots.py
@@ -210,9 +210,9 @@ def plot_reward_vs_time(
         X = []
         Y = []
         for traj in trajs_list:
-            T = len(traj.rews)
+            T = len(traj.acts)
             X.extend(range(T))
-            dones = np.zeros(T, dtype=np.bool)
+            dones = np.zeros(T, dtype=bool)
             dones[-1] = True
             rews = reward_fn(traj.obs[:-1], traj.acts, traj.obs[1:], dones)
             Y.extend(rews)

